401-react-clickable-grid/401-submit/test/grid-view.spec.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shallow, render } from 'enzyme';
+import sinon from 'sinon';
+import { expect } from 'chai';
+
+import GridComponent from '../src/js/grid-view';
+
+describe('<GridElement />', () => {
```

Comment:
have a test is good. You should also test things that are outside of the normal range